README.md

```diff
@@ -134,13 +134,19 @@ $ docker run -p 8080:80 -e SERVER_NAME=myhost my-modsec
 | METRICS_DENY_FROM  | A string indicating a range of IP adresses that cannot access the metrics (Default: `all`) | 
 | METRICSLOG  | A string value indicating the location of metrics log file (Default: `/dev/null`) | 
 | PORT  | An integer value indicating the port where the webserver is listening to (Default: `80`) | 
-| PROXY_SSL_CERT_KEY  | A string value indicating the path to the server PEM-encoded private key file (Default: `/etc/nginx/conf/server.key`) | 
+| NGINX_ALWAYS_TLS_REDIRECT | A string value indicating if http should redirect to https (Default: `off`) | 
```

Comment:
One small comment here: I did setup using `1` and `On`, and didn't worked. I've assumed that anything set was going to be catched properly, but turn out it didn't.

I would add then after the `(Default: off)`, something like `Allowed values: "on", "off"`, so we don't get other people frustrated :D